core/lib/web3_decl/src/namespaces/eth.rs

```diff
@@ -87,7 +87,10 @@ pub trait EthNamespace {
     ) -> RpcResult<Option<U256>>;
 
     #[method(name = "getBlockReceipts")]
-    async fn get_block_receipts(&self, block_id: BlockId) -> RpcResult<Vec<TransactionReceipt>>;
+    async fn get_block_receipts(
+        &self,
+        block_id: BlockId,
+    ) -> RpcResult<Option<Vec<TransactionReceipt>>>;
```

Comment:
AFAIU, "canonical" Ethereum Web3 implementations return `null` for non-existing blocks, so I've changed this method to match this behavior.